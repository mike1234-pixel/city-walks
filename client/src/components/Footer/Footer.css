.footer {
  background-color: #212121;
}

/* removes bootstrap margin on footer */
.mt-4 {
  margin-top: 0 !important;
}
