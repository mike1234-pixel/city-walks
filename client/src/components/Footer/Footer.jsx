import React from 'react'
import { Link } from "react-router-dom"
import { MDBCol, MDBContainer, MDBRow, MDBFooter } from 'mdbreact'
import './Footer.css'

const Footer = () => {
  return (
    <MDBFooter className="font-small pt-4 mt-4 footer">
      <MDBContainer fluid className="text-center text-md-left">
        <MDBRow>
          <MDBCol md="6">
            <h5 className="title">City Walks is...</h5>
            <p>
              (City Walks description)
            </p>
          </MDBCol>
          <MDBCol md="3">
            <h5 className="title">Site Links</h5>
            <ul>
            <li className="list-unstyled">
                <Link to="/">Home</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/cities">Cities</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/walks">Walks</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/about">About</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/contact">Contact</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/login">User Login</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/admin">Admin portal</Link>
              </li>
            </ul>
          </MDBCol>
          <MDBCol md="3">
          <h5 className="title">External Links</h5>
          <ul>
          <li className="list-unstyled">
                <Link to="/login">User Login</Link>
              </li>
              <li className="list-unstyled">
                <Link to="/admin">Admin portal</Link>
              </li>
            </ul>
          </MDBCol>
        </MDBRow>
      </MDBContainer>
      <div className="footer-copyright text-center py-3">
        <MDBContainer fluid>
          &copy; {new Date().getFullYear()} Copyright
        </MDBContainer>
      </div>
    </MDBFooter>
  );
}

export default Footer;