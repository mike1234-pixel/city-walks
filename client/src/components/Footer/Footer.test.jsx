import renderer from 'react-test-renderer'
import Footer from './Footer'

// snapshot test
it('Footer renders correctly', () => {  

  const tree = renderer
    .create(<Footer/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});