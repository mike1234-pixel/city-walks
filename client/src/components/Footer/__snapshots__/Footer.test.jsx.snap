// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="page-footer blue font-small pt-4 mt-4"
  data-test="footer"
>
  <div
    className="container-fluid text-center text-md-left"
    data-test="container"
  >
    <div
      className="row"
      data-test="row"
    >
      <div
        className="col-md-6"
        data-test="col"
      >
        <h5
          className="title"
        >
          Footer Content
        </h5>
        <p>
          Here you can use rows and columns here to organize your footer content.
        </p>
      </div>
      <div
        className="col-md-6"
        data-test="col"
      >
        <h5
          className="title"
        >
          Links
        </h5>
        <ul>
          <li
            className="list-unstyled"
          >
            <a
              href="#!"
            >
              Link 1
            </a>
          </li>
          <li
            className="list-unstyled"
          >
            <a
              href="#!"
            >
              Link 2
            </a>
          </li>
          <li
            className="list-unstyled"
          >
            <a
              href="#!"
            >
              Link 3
            </a>
          </li>
          <li
            className="list-unstyled"
          >
            <a
              href="#!"
            >
              Link 4
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="footer-copyright text-center py-3"
  >
    <div
      className="container-fluid"
      data-test="container"
    >
      Â© 
      2020
       Copyright: 
      <a
        href="https://www.mdbootstrap.com"
      >
         MDBootstrap.com 
      </a>
    </div>
  </div>
</footer>
`;
